session/ssh_session.go

```diff
@@ -241,9 +247,17 @@ func (s *SSHSession) authFromToken(c ssh.ConnMetadata, pass []byte) (*ssh.Permis
 	if backendID == "" {
 		return nil, errors.New("token rejected")
 	}
+
+	// assume false if not set
+	requiresClientAuth, err := s.store.BackendRequiresClientAuth(backendID)
+	if err != nil && err != redis.ErrNil {
```

Comment:
I don't think we should be checking the error type here `err != redis.ErrNil`? If I'm reading the `BackendRequiresClientAuth` code right, it  handles checking for `redis.ErrNil`. So shouldn't need this here if I'm understanding that properly

Reply:
yup, i must have rolled changes in there later, will simplify that